python/paddle/fluid/contrib/mixed_precision/decorator.py

```diff
@@ -297,13 +326,47 @@ def apply_gradients(self, params_grads):
         if not self._use_dynamic_loss_scaling and self._init_loss_scaling == 1.0:
             return self._optimizer.apply_gradients(params_grads)
 
+        if self._supports_check_nan_inf():
+            self._optimizer._set_scale(self._loss_scaling)
+            optimize_ops = self._optimizer.apply_gradients(params_grads)
+            found_inf = self._optimizer._found_inf
+            self._add_dynamic_loss_scaling(params_grads, found_inf)
```

Comment:
Not important, but it seems params_grads is not used in `_add_dynamic_loss_scaling` in this case.

Reply:
Yes. But not quite important.